### PR TITLE
Ensure we sort the results

### PR DESCRIPTION
**Pull request by SimonRichardson**
The following ensures we sort the results to get consistent results

### CODE REVIEW

**Comment by mitechie** [comment]
results of what? :P

**Comment by SimonRichardson** [comment]
@mitechie so if you have a lot of commands and the distance to the string is the same, because we stored them in map, the results will be in a seemingly random order. So it's best to sort them in an array so the results are at least consistent.

File: supercommand.go [hunk 1]
```diff
@@ -518,17 +518,30 @@ func (c *SuperCommand) Run(ctx *Context) error {
...
+	matches := make([]Indexed, 0, len(c.subcmds))
 	for cmdName := range c.subcmds {
-		matches[cmdName] = levenshteinDistance(name, cmdName)
+		matches = append(matches, Indexed{
+			Name:  cmdName,
+			Value: levenshteinDistance(name, cmdName),
+		})
 	}
 
+	// ensure that we're dealing with the same command names, so we can give
```
Comment (mitechie) [inline supercommand.go]: start with a capital and end with punctuation please

File: supercommand.go [hunk 2]
```diff
@@ -518,17 +518,30 @@ func (c *SuperCommand) Run(ctx *Context) error {
...
+
 	var matchedName string
 	matchedValue := math.MaxInt64
-	for cmdName, value := range matches {
-		if value < matchedValue {
-			matchedName = cmdName
-			matchedValue = value
+	for _, indexed := range matches {
+		if indexed.Value < matchedValue {
+			matchedName = indexed.Name
```
Comment (mitechie) [inline supercommand.go]: but no tests?

**Comment by SimonRichardson** [comment]
$$merge$$

**Pull request merged**